src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy

```diff
@@ -586,6 +586,43 @@ class LagPortSpec extends HealthCheckSpecification {
         lagPort && northboundV2.deleteLagLogicalPort(sw.dpId, lagPort)
     }
 
+    @Tidy
+    def "Unable decrease bandwidth on LAG port lower than connected flows bandwidth sum"() {
+        given: "Flows on a LAG port with switch ports"
+        def switchPair = topologyHelper.getSwitchPairs().first()
+        def testPorts = topology.getAllowedPortsForSwitch(switchPair.src).takeRight(2).sort()
+        assert testPorts.size > 1
+        def maximumBandwidth = testPorts.sum { northbound.getPort(switchPair.src.dpId, it).maxSpeed }
```

Comment:
I would suggest using `port.curr_speed`. lag port bandwidth is equal to curr sped sum of physical ports. Mostly, in our cases curr speed is equal to max speed. But if you have 2 ports with curr speed 10Gb, and max speed 60Gb, you would not be able to create a flow. Correct me, if I'm wrong.

Reply:
cnahged from .maxSpeed to .currentSpeed